plugins/modules/mso_schema_site_anp_epg_bulk_staticport.py

```diff
@@ -434,35 +432,28 @@ def main():
                 new_leaf.update(microSegVlan=primary_micro_segment_vlan)
 
             # validate and append staticports to staticport_list if path variable is different
-            if path in unique_paths:
-                mso.fail_json(msg="Path should be unique for each Static Port")
+            if portpath in unique_paths:
+                mso.fail_json(msg="Each leaf in a pod of a static port should have an unique path.")
             else:
-                unique_paths.append(path)
+                unique_paths.append(portpath)
                 staticport_list.append(new_leaf)
 
-    # If payload is empty, anp and EPG already exist at site level
-    if not payload:
-        op_path = ports_path
-        payload = staticport_list
-    else:
-        # If anp already exists at site level
-        if "anpRef" not in payload:
-            payload["staticPorts"] = staticport_list
+        # If payload is empty, anp and EPG already exist at site level
+        if not payload:
+            payload = staticport_list
         else:
```

Comment:
Do we still need this else? Think the if below can be elif, which makes it less nested

Reply:
sure, changed and tested for both comments.